editor/plugins/material_editor_plugin.cpp

```diff
@@ -37,6 +37,19 @@
 #include "scene/resources/particles_material.h"
 #include "scene/resources/sky_material.h"
 
+void MaterialEditor::gui_input(const Ref<InputEvent> &p_event) {
+	ERR_FAIL_COND(p_event.is_null());
+
+	Ref<InputEventMouseMotion> mm = p_event;
+	if (mm.is_valid() && (mm->get_button_mask() & MouseButton::MASK_LEFT) != MouseButton::NONE) {
```

Comment:
I'd allow rotation with both the left mouse button *and* middle mouse button. This way, you can also use the middle mouse button like you'd do in the 3D editor viewport.

Reply:
Seems like  good usability but would be better as a future PR which adds this usability across all editors :)